app/templates/components/forms/admin/settings/system/storage-option.hbs

```diff
@@ -0,0 +1,90 @@
+<h3 class="ui header">
+  {{t 'Storage'}}
+  <div class="sub header">
+    {{t 'Where to store data ?'}}
+  </div>
+</h3>
+<div class="field">
+  {{ui-radio label=(t 'Local') current ='local' name='storage' value='local' onChange=(action (mut selectedStorage))}}
+  <span class="small text">
+    ({{t 'stored in'}} /static/media/folder)
+  </span>
+</div>
+<div class="field">
+  {{ui-radio label=(t 'Google Storage') name='storage' value='google' onChange=(action  (mut selectedStorage))}}
+  <span class="small text">
+    ({{t 'See'}}
+    <a href="https://cloud.google.com/storage/docs/migrating#defaultproj" target="_blank" rel="noopener nofollow">
+      {{t 'here'}}
+    </a>
+    {{t 'on how to obtain these keys.'}})
+  </span>
+</div>
+<div class="grouped fields {{if (not-eq selectedStorage 'google') 'disabled'}}">
+  <div class="field">
+    <label>
+      {{t 'Bucket name'}}
+    </label>
+    {{input type='text' name='google_storage_bucket_name' disabled=(not-eq selectedStorage 'google')}}
+  </div>
+  <div class="field">
+    <label>
+      {{t 'Interoperable access key'}}
+    </label>
+    {{input type='text' name='google_storage_access_key' disabled=(not-eq selectedStorage 'google')}}
+  </div>
+  <div class="field">
+    <label>
+      {{t 'Interoperable access secret'}}
+    </label>
+    {{input type='text' name='google_storage_access_secret' disabled=(not-eq selectedStorage 'google')}}
+  </div>
+</div>
+<div class="ui hidden divider"></div>
+<div class="field">
+  {{ui-radio label=(t 'Amazon S3') name='storage' value='S3' onChange=(action (mut selectedStorage))}}
+  <span class="small text">
+    ({{t 'See'}}
+    <a href="http://docs.aws.amazon.com/AWSSimpleQueueService/latest/SQSDeveloperGuide/sqs-getting-started.html" target="_blank" rel="noopener nofollow">
+      {{t 'here'}}
+    </a>
+    {{t 'on how to obtain these keys.'}})
+  </span>
+</div>
+<div class="grouped fields {{if (not-eq selectedStorage 'S3') 'disabled'}}">
```

Comment:
I have added grouped fields class here, just to ensure that fields are disabled via CSS as well to give the semantic-ui disabled look.